features/step_definitions/websocket-steps.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const should = require('should');
+const { Given, Then, When } = require('cucumber');
+const ws = require('ws');
+
+Given('I open a new local websocket connection', async function () {
+  this.props.client = new ws('ws://localhost:7512');
+  const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+  await delay(1000);
+  this.props.client.on('message', data => {
+    this.props.result = JSON.parse(data);
+  });
+});
+
+When('I send the message {string} to Kuzzle through websocket', function (message) {
+  this.props.client.send(message);
+});
+
+Then('I should receive a response message from Kuzzle through websocket matching:', function (dataTable) {
```

Comment:
We may not have received the answer here

Reply:
Kuzzle sends automatically a message when it received one, I'm just checking if it's the good one

Reply:
Yes but when this step is executed, maybe we still didn't receive a response from Kuzzle